### PR TITLE
Add lichen-markdown to yunohost apps catalog 

### PR DESCRIPTION
**Pull request by mhfowler**
Lichen-markdown is an ultra-light-weight permacomputing-inspired CMS. 

The yunohost package is here: https://github.com/YunoHost-Apps/lichenmarkdown_ynh

More information about the upstream project is here: https://lichen.commoninternet.net

### CODE REVIEW

**Comment by orhtej2** [comment]
With [this line in place](https://github.com/YunoHost-Apps/lichenmarkdown_ynh/blob/main/tests.toml#L9) the app will remain at lvl0 forever and will be eventually marked as broken in the catalog.

**Comment by orhtej2** [comment]
Testing naive fix here: YunoHost-Apps/lichenmarkdown_ynh#3.

**Comment by orhtej2** [comment]
Their tagging policy is questionable at best: https://codeberg.org/ukrudt.net/lichen-markdown/releases/tag/v1.0.0

> previous releases were deleted due to containing large files we wanted to keep out of the git repository, so all commits were rebased into one commit, and other branches and tags were pruned with a little somewhat questionable git magic

**Comment by mhfowler** [comment]
@orhtej2 wrt the tagging, one of the goals of the project is to provide a CMS in as few KB as possible, so that's why there is unusual attention to removing all unneeded files

File: apps.toml [hunk 1]
```diff
@@ -2237,6 +2237,12 @@ level = 6
 state = "working"
 url = "https://github.com/YunoHost-Apps/librex_ynh"
 
+[lichenmarkdown]
+added_date = 1732492800 # 2024/11/25
```
Comment (ericgaspar) [inline apps.toml:2241]:
```suggestion
```

**Review by ericgaspar — APPROVED** [review]

**Pull request merged**